web/templates/compose.hbs

```diff
@@ -70,7 +70,10 @@
                             <div class="topic-marker-container">
                                 <i class="fa fa-angle-right" aria-hidden="true"></i>
                             </div>
-                            <input type="text" class="recipient_box" name="stream_message_recipient_topic" id="stream_message_recipient_topic" maxlength="{{ max_topic_length }}" value="" placeholder="{{t 'Topic' }}" autocomplete="off" tabindex="0" aria-label="{{t 'Topic' }}" />
+                            <div id="compose_recipient_box">
```

Comment:
This is model HTML for how to handle these kinds of "illusion" inputs, where one interactive element (a button) appears to sit inside another (an input). Nice work!